components/Navigation/Navigation.js

```diff
@@ -7,13 +7,20 @@ import {
   NavButton,
 } from "./Navigation.styled";
 import { signIn, signOut, useSession } from "next-auth/react";
+import { navigationEntries } from "./navigationEntries";
 
 export default function Navigation() {
   const [isOpen, setIsOpen] = useState(false);
   const { data: session } = useSession();
   const toggleMenu = () => {
     setIsOpen(!isOpen);
   };
+  let navigationLinks = navigationEntries;
+  if (!session) {
+    navigationLinks = navigationEntries.filter(
+      (entry) => entry.requireSession === false
+    );
+  }
```

Comment:
We can make the code more readable by utilizing a ternary operator directly when defining navigationLinks. This approach eliminates the need for a mutable variable and a separate conditional block. 

```suggestion
const navigationLinks = session 
    ? navigationEntries 
    : navigationEntries.filter(entry => !entry.requireSession);
```